compiler/rustc_attr_data_structures/src/attributes.rs

```diff
@@ -160,37 +152,49 @@ impl Deprecation {
 #[derive(Clone, Debug, HashStable_Generic, Encodable, Decodable, PrintAttribute)]
 pub enum AttributeKind {
     // tidy-alphabetical-start
+    /// Represents [`#[rustc_allow_const_fn_unstable]`](<https://github.com/rust-lang/rust/issues/69399>).
```

Comment:
Nit: I *think* these need to be (no `<>`)

```suggestion
    /// Represents [`#[rustc_allow_const_fn_unstable]`](https://github.com/rust-lang/rust/issues/69399).
```